{"name": "test_registration_001", "status": "failed", "statusDetails": {"message": "AssertionError: assert True == False", "trace": "self = <testCases.test_createAccount.Test_CreateAccount object at 0x00000263E2B77690>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"217ed79ea2e610846b8f1ffc9fdf0676\")>\n\n    @pytest.mark.sanity\n    def test_registration_001(self, setup):\n        self.driver = setup\n        self.log.info(\"TestCase test_registration_001 is started\")\n        self.log.info(\"invoking the browser\")\n        self.driver.get(self.url)\n        self.log.info(\"opening URL\")\n        self.rp = UserRegistrationClass(self.driver)\n        self.rp.link_registration()\n        self.log.info(\"clicking on registration link\")\n        self.rp.firstname(\"Lukesh\")\n        self.log.info(\"entering firstname\")\n        self.rp.lastname(\"Ade\")\n        self.log.info(\"entering lastname\")\n        self.email = random_generator() + \"@gmail.com\"\n        self.rp.setEmail(self.email)\n        self.log.info(\"entering the email\")\n        self.rp.password(self.password)\n        self.log.info(\"entering the password\")\n        self.rp.confirm_password(self.password)\n        self.log.info(\"entering the confirm password\")\n        self.rp.submit_button()\n        self.log.info(\"clicking submit button\")\n        self.log.info(\"saving customer info\")\n        self.log.info(\"user_registration validation start\")\n        self.msg = self.driver.find_element(By.XPATH, \"/html/body/div[1]/main/div[1]/div[2]/div/div/div\").text\n        if 'Thank you for registering with Main Website Store.' in self.msg:\n            assert True == True\n            self.driver.save_screenshot(\".\\\\Screenshot\\\\test_registration_001_pass.png\")\n            self.log.info(\"test_registration_001 is pass\")\n        else:\n            self.driver.save_screenshot(\".\\\\Screenshot\\\\test_registration_001_fail.png\")\n            self.log.info(\"test_registration_001 is fail\")\n>           assert True == False\nE           assert True == False\n\ntestCases\\test_createAccount.py:51: AssertionError"}, "attachments": [{"name": "log", "source": "9ef355e1-0c46-4656-acea-b274f20858a4-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "0825c1a2-f08b-4809-a709-2b9d7a0694c4-attachment.txt", "type": "text/plain"}], "start": 1691057665993, "stop": 1691057682732, "uuid": "80855d6f-a472-407f-9802-3f8c6f6d2776", "historyId": "e1a3a18cc3b89066322e973555f917e3", "testCaseId": "e1a3a18cc3b89066322e973555f917e3", "fullName": "testCases.test_createAccount.Test_CreateAccount#test_registration_001", "labels": [{"name": "tag", "value": "sanity"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_createAccount"}, {"name": "subSuite", "value": "Test_CreateAccount"}, {"name": "host", "value": "DESKTOP-3ABMBD4"}, {"name": "thread", "value": "5152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_createAccount"}]}